docs/Learning-Environment-Examples.md

```diff
@@ -56,7 +56,7 @@ If you would like to contribute environments, please see our
   * Vector Action space: (Continuous) Size of 2, with one value corresponding to
     X-rotation, and the other to Z-rotation.
   * Visual Observations: None.
-* Reset Parameters: None
+* Reset Parameters: Three, corresponding to ball size, ball mass and gravity.
```

Comment:
`Reset Parameters:
- Ball Size:
- Ball Mass:
- Gravity:`

(use actual names and add description)